docs/how-tos/using-events.rst

```diff
@@ -0,0 +1,113 @@
+Using Open edX Events
+=====================
+
+How to use
+----------
+
+Using openedx-events in your code is very straight forward. We can consider the
+two possible cases, sending or receiving an event.
+
+Receiving events
+^^^^^^^^^^^^^^^^
+
+This is one of the most common use cases for plugins. The edx-platform will send
+and event and you want to react to it in your plugin.
+
+For this you need to:
+
+1. Include openedx-events in your dependencies.
+2. Connect your receiver functions to the signals being sent.
+
+Connecting signals can be done using regular django syntax:
+
+.. code-block:: python
+
+    from openedx_events.learning.signals import STUDENT_REGISTRATION_COMPLETED
+
+    @receiver(STUDENT_REGISTRATION_COMPLETED)
+    def your_receiver_function(**kwargs):
+        # your implementation here
+
+
+Or at the apps.py
+
+.. code-block:: python
+
+    {
+        "signals_config": {
+            "lms.djangoapp": {
+                "relative_path": "your_module_name",
+                "receivers": [
+                    {
+                        "receiver_func_name": "your_receiver_function",
+                        "signal_path": "openedx_events.learning.signals.STUDENT_REGISTRATION_COMPLETED",
+                    },
+                ],
+            }
+        }
+    }
+
+
+In case you are listening to an event in the edx-platform repo, you can directly
+use the django syntax since the apps.py method will not be available without the
+plugin.
```

Comment:
Should we warn that any non-trivial work should be done in an asynchronous worker task? I think @ormsbee favored any work, because even trivial work may slowly become non-trivial over time. Again, not a blocker, but if we could agree and add sooner than later, that would be nice.

Reply:
Yes, I agree that we should encourage the use of that pattern. I added a short warning, what do you think?